### ISSUE TITLE
Extracting from html file is broken

### ISSUE THREAD

**Issue by vbraun** [Polyconseil/easygettext#73]
Extracting html files works with easygettext@2.7.0, but not with 2.8.0 or later. It seems like it is fed into acorn which can only deal with js, not html:
~~~~
$ npx gettext-extract --output /tmp/dictionary.pot /var/tmp/test.html
[easygettext] extracting: '/var/tmp/test.html
[easygettext] could not read: '/var/tmp/test.html
Trace: { SyntaxError: Unexpected token (1:0)
    at Object.pp$4.raise (/home/vbraun/node_modules/acorn/dist/acorn.js:2825:15)
    at Object.pp.unexpected (/home/vbraun/node_modules/acorn/dist/acorn.js:689:10)
    at Object.pp$3.parseExprAtom (/home/vbraun/node_modules/acorn/dist/acorn.js:2270:12)
    at Object.parseExprAtom (/home/vbraun/node_modules/acorn-stage3/node_modules/acorn-dynamic-import/lib/index.js:75:117)
    at Object.parseExprAtom (/home/vbraun/node_modules/acorn-import-meta/index.js:17:75)
    at Object.pp$3.parseExprSubscripts (/home/vbraun/node_modules/acorn/dist/acorn.js:2089:21)
    at Object.pp$3.parseMaybeUnary (/home/vbraun/node_modules/acorn/dist/acorn.js:2066:19)
    at Object.parseMaybeUnary (/home/vbraun/node_modules/acorn-private-class-elements/index.js:112:29)
    at Object.pp$3.parseExprOps (/home/vbraun/node_modules/acorn/dist/acorn.js:2010:21)
    at Object.pp$3.parseMaybeConditional (/home/vbraun/node_modules/acorn/dist/acorn.js:1993:21)
    at Object.pp$3.parseMaybeAssign (/home/vbraun/node_modules/acorn/dist/acorn.js:1968:21)
    at Object.pp$3.parseExpression (/home/vbraun/node_modules/acorn/dist/acorn.js:1933:21)
    at Object.pp$1.parseStatement (/home/vbraun/node_modules/acorn/dist/acorn.js:877:47)
    at Object.parseStatement (/home/vbraun/node_modules/acorn-stage3/node_modules/acorn-dynamic-import/lib/index.js:63:118)
    at Object.parseStatement (/home/vbraun/node_modules/acorn-import-meta/index.js:38:22)
    at Object.pp$1.parseTopLevel (/home/vbraun/node_modules/acorn/dist/acorn.js:746:23) pos: 0, loc: Position { line: 1, column: 0 }, raisedAt: 1 }
    at /home/vbraun/node_modules/easygettext/src/extract-cli.js:85:13
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/vbraun/node_modules/easygettext/src/extract-cli.js:57:7)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at findNodeScript.then.existing (/usr/lib/node_modules/npm/node_modules/libnpx/index.js:268:14)
~~~~
My use case is defining vue components in separate files, so the vue file is only `<template src="./filename.html"></template>`. To extract I'm calling gettext-extract on the html file.

**Comment by vperron** [comment]
I have this tab open since a good whole week :( Sorry I haven't had any time yet to look into it...

Do you think you could corner what's going on ? The codebase is pretty small !

Thanks a lot,

**Comment by lburg** [comment]
@vbraun Should be fixed with version 2.10!
